libraries/service/src/main/java/org/holochain/androidserviceruntime/service/HolochainService.kt

```diff
@@ -88,30 +79,35 @@ class HolochainService : Service() {
         Log.d(logTag, "onStartCommand")
 
         when (intent?.action) {
-            ACTION_START -> {
+            HolochainServiceIntentActions.ACTION_START -> {
                 Log.d(
                     logTag,
                     "onStartCommand ACTION_START",
                 )
 
-                // Enable conductor to autostart on system boot
-                getAutostartConfigManager().enable()
+                // Get config from intent
+                val config = intent.extras?.getParcelable<RuntimeNetworkConfigFfiParcel>("config")
+                if (config != null) {
+                    // Enable conductor to autostart on system boot
+                    getAutostartConfigManager().enable(config.inner)
 
-                // Start service
-                startForeground()
+                    // Start service
+                    startForeground(config.inner)
+                }
             }
-            ACTION_BOOT_COMPLETED -> {
+            HolochainServiceIntentActions.ACTION_BOOT_COMPLETED -> {
                 Log.d(
                     logTag,
                     "onStartCommand ACTION_BOOT_COMPLETED",
                 )
 
                 // If autostart is enabled, start service
-                if (getAutostartConfigManager().isEnabled()) {
-                    startForeground()
+                val config = getAutostartConfigManager().getEnabledConfig()
```

Comment:
When auto-starting starting the service, we read the runtime network config from the autostart manager.